nixos/modules/services/x11/desktop-managers/phosh.nix

```
@@ -78,7 +78,7 @@ let
         description = ''
           Display scaling factor.
         '';
-        type = types.nullOr types.ints.unsigned;
+        type = types.nullOr (types.either types.ints.unsigned types.float);
```

A scaling factor of 0 doesn't make sense, does it?
```suggestion
        type = types.nullOr (
          types.addCheck
          (types.either types.int types.float)
          (x : x > 0)
        ) // {
          description = "null or positive integer or float"
        };
```

---

applied and re-deployed to my pinephone just to be sure :-)